src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -296,6 +297,22 @@ public int getNumResidents() {
 		return residents.size();
 	}
 
+	public int getNumResidentsOnline() {
+		int onlineResidents = 0;
+		for(Resident resident: getResidents()) {
+			if(resident.isNPC()) {
+				onlineResidents++;
+				continue;
+			}
+			
+			Player player = Bukkit.getServer().getPlayer(resident.getName());
+			if(player.isOnline()) {
+				onlineResidents++;
+			}
+		}
+		return onlineResidents;
+	}
+	
```

Comment:
This needs to be removed, we already have a method `TownyAPI#getOnlineResidents()` that does this.